patches/chrome-android-java-src-org-chromium-chrome-browser-sync-settings-ManageSyncSettings.java.patch

```diff
@@ -1,13 +1,80 @@
 diff --git a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/ManageSyncSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/ManageSyncSettings.java
-index 4a7c610cec562ed067a86629b8a6f5d0b8758ddf..e5b2f7a726b7efe4d3f5dfe2322a1d375e49f631 100644
+index 4a7c610cec562ed067a86629b8a6f5d0b8758ddf..4c39c4ed51ec0299d65637c442d7119277615d48 100644
 --- a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/ManageSyncSettings.java
 +++ b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/ManageSyncSettings.java
-@@ -64,7 +64,7 @@ import java.util.Set;
-  * Settings fragment to customize Sync options (data types, encryption). Can be accessed from
-  * {@link SyncAndServicesSettings}.
-  */
--public class ManageSyncSettings extends PreferenceFragmentCompat
-+public class ManageSyncSettings extends org.chromium.chrome.browser.settings.BravePreferenceFragment
-         implements PassphraseDialogFragment.Listener, PassphraseCreationDialogFragment.Listener,
-                    PassphraseTypeDialogFragment.Listener, Preference.OnPreferenceChangeListener,
-                    ProfileSyncService.SyncStateChangedListener,
+@@ -116,15 +116,15 @@ public class ManageSyncSettings extends PreferenceFragmentCompat
+     private PreferenceCategory mSyncingCategory;
+ 
+     private ChromeSwitchPreference mSyncEverything;
+-    private CheckBoxPreference mSyncAutofill;
```

Comment:
cc @bridiver for patching check

Reply:
This is definitely a problem and can be done with asm. I'll take a look at it